### DISCUSSION TITLE
AddConsoleExporter() method not working in .NET6 WPF application

### DISCUSSION THREAD

**Discussion by SanyP23** [open-telemetry/community#1606] [Community Discussions]
Hi,

AddConsoleExporter() method not working in .NET6 WPF or .NET6 winforms application. It works upto .NET framework 4.7.2, but for .NET 6 it only generates log if it is a console app. Is it not supported for .NET 6 WPF application?

**Comment by trask** [comment]
Hi @SanyP23! I'd suggest asking this in https://github.com/open-telemetry/opentelemetry-dotnet
